### PR TITLE
include new lang flag guidance issue #1599

### PR DESCRIPTION
**Pull request by fozy81**

### CODE REVIEW

**Comment by fozy81** [comment]
Hi @locks - first draft for including `--lang` guidance. Please review, feedback welcome.

File: guides/release/accessibility/application-considerations.md [hunk 1]
```diff
@@ -7,7 +7,16 @@ Declaring the language of the HTML document allows users to better understand yo
 > Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. As a result, users with disabilities will be better able to understand the content.
 > [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
 
-A primary language should be defined on the `<html>` element's `lang` attribute. For existing Ember apps, a developer may edit the `index.html` file or leverage [ember-intl](https://github.com/ember-intl/ember-intl).
+The `--lang` option on the `ember new` command sets the language for a new application.
```
Comment (locks) [inline guides/release/accessibility/application-considerations.md:10]:
```suggestion
A primary language should be defined on the `<html>` element's `lang` attribute. For new apps, you can use the `--lang` option on the `ember new` command to set the language for a new application.
```

**Review by locks — CHANGES_REQUESTED** [review]
Small tweak inline. What do you think?

**Comment by fozy81** [comment]
@locks Agree, that flows better. Thanks for your suggestion.

**Review by locks — APPROVED** [review]
Let's go!

**Comment by locks** [comment]
Thanks for the help :) I'm waiting on CI to then merge your work.

**Review by ijlee2 — APPROVED** [review]
Thank you for adding an explanation of how to create an app with `lang` set up!

It seems that CI didn't work properly last time. I reran CI and think we can ignore the 2 jobs (`CI / Test app` and `CI / Test node-tests`) that had failed.

**Pull request merged**